depends/patches/qt/guix_cross_lib_path.patch

```diff
@@ -1,17 +0,0 @@
-Facilitate guix building with CROSS_LIBRARY_PATH
```

Comment:
In 54d81cab53a07fc47ddd38d3be0384651d785558: You're removing this patch, but looking at the upstream source for 6.7.3, it's not immediately clear where this was fixed upstream, and the commit doesn't explain why it's no-longer needed. Can you elaborate? I'm wondering if new patches you are introducing are just a result of deleting this one.

Reply:
> In [54d81ca](https://github.com/bitcoin/bitcoin/commit/54d81cab53a07fc47ddd38d3be0384651d785558): You're removing this patch, but looking at the upstream source for 6.7.3, it's not immediately clear where this was fixed upstream, and the commit doesn't explain why it's no-longer needed. Can you elaborate?

The patched files are part of the legacy qmake-based build system and are unrelated to the new CMake-based build system.

> I'm wondering if new patches you are introducing are just a result of deleting this one.

I had the same thought while working on this PR. However, I couldn’t find corresponding CMake code that performs a similar cleanup.